File: surf-cloud-core/src/main/java/dev/slne/surf/cloud/core/netty/common/source/NettyServerSourceImpl.java
package dev.slne.surf.cloud.core.netty.common.source;

import dev.slne.surf.cloud.api.netty.source.NettyServerSource;
import dev.slne.surf.cloud.core.netty.AbstractNettyBase;

public class NettyServerSourceImpl extends AbstractProxiedNettySource<NettyServerSource> implements
    NettyServerSource {

  public NettyServerSourceImpl(AbstractNettyBase<?, ?, NettyServerSource> nettyBase) {
    super(nettyBase);
  }
}

File: surf-cloud-core/src/main/kotlin/dev/slne/surf/cloud/core/netty/common/source/NettyServerSourceImpl.kt
package dev.slne.surf.cloud.core.netty.common.source

import dev.slne.surf.cloud.api.netty.source.NettyServerSource
import dev.slne.surf.cloud.core.netty.AbstractNettyBase

class NettyServerSourceImpl(nettyBase: AbstractNettyBase<*, *, NettyServerSource>) :
    AbstractProxiedNettySource<NettyServerSource>(nettyBase), NettyServerSource
